extensions/sql-database-projects/package.json

```diff
@@ -251,6 +260,11 @@
           "when": "view == sqlDatabaseProjectsView && viewItem == databaseProject.itemType.folder || viewItem == databaseProject.itemType.file",
           "group": "9_dbProjectsLast@2"
         },
+        {
+          "command": "sqlDatabaseProjects.openContainingFolder",
+          "when": "view == sqlDatabaseProjectsView && viewItem == databaseProject.itemType.project",
+          "group": "9_dbProjectsLast@8"
```

Comment:
Just a question for my knowledge: How does the group value effect the visibility/sequence location of command? Or is it more about the sequence followed in the code that dictates its placement?

Reply:
For `9_dbProjectsLast@8`, it means:

* the group is named "dbProjectsLast"
* the group is ordered as number 9 in the list of groups
* _within the group_, this particular command is ordered as number 8

Caveat: I say "ordered as number _X_" instead of "is _X_th" because you can skip numbers to leave places for other menus.  I used 9 here and skipped 5 through 8 to keep this one last as more groups get added.

If those numbers are not specified, the groups are ordered alphabetically, then commands within the groups are alphabetically.